### PR TITLE
chore: naming

### PR DESCRIPTION
**Pull request by tanguyfalconnet**
## Why

A few renaming to match app name everywhere, and update readme.

### CODE REVIEW

**Review by arthurzinck — APPROVED** [review]
:1st_place_medal:

File: main.go [hunk 1]
```diff
@@ -64,32 +64,32 @@ var (
 
 	metric_present_1_day = promauto.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "alert_validator_present_over_1days",
+			Name: "alertsvalidator_present_over_1day",
 			Help: "does the alert's metrics contains data point over 1 day",
```
Comment (christian-vdz) [inline main.go:68]:
```suggestion
			Help: "does the alert's metrics contain data points over 1 day",
```

File: main.go [hunk 2]
```diff
@@ -64,32 +64,32 @@ var (
...
 			"tenant",
-			"alert_name",
+			"alertname",
 		},
 	)
 	metric_present_30_days = promauto.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "alert_validator_present_over_30days",
+			Name: "alertsvalidator_present_over_30days",
 			Help: "does the alert's metric contains data point over the last 30 days",
```
Comment (christian-vdz) [inline main.go:78]:
```suggestion
			Help: "does the alert's metric contain data points over the last 30 days",
```

File: main.go [hunk 3]
```diff
@@ -64,32 +64,32 @@ var (
...
 			"tenant",
-			"alert_name",
+			"alertname",
 		},
 	)
 	metric_present_90_days = promauto.NewGaugeVec(
 		prometheus.GaugeOpts{
-			Name: "alert_validator_present_over_90days",
+			Name: "alertsvalidator_present_over_90days",
 			Help: "does the alert's metric contains data point over the last 90 days",
```
Comment (christian-vdz) [inline main.go:88]:
```suggestion
			Help: "does the alert's metric contain data points over the last 90 days",
```

**Pull request merged**